azure/template.json

```diff
@@ -841,7 +841,7 @@
                         "value": "AZURE_IP-"
                     },
                     "ipAddresses": {
-                        "value": "[reference('app-service').outputs.possibleOutboundIpAddresses.value]"
+                      "[concat(reference('app-service').outputs.possibleOutboundIpAddresses.value, array(reference('container-instances-worker').outputs.containerIPv4Address.value))]"
```

Comment:
This line should read `"value": "[concat(reference('app-service').outputs.possibleOutboundIpAddresses.value, array(reference('container-instances-worker').outputs.containerIPv4Address.value))]"`

Reply:
Fixed, also added a test that would've caught this:

https://github.com/DFE-Digital/apply-for-postgraduate-teacher-training/pull/553/commits/1f05cae2b5a29c1b82eb95b19707d40e76d64bb2